addons/portal/controllers/portal.py

```diff
@@ -175,51 +288,22 @@ def counters(self, counters, **kw):
     def home(self, **kw):
         values = self._prepare_portal_layout_values()
         values.update(self._prepare_portal_overview_values())
+        values['company'] = request.env.company
```

Comment:
Don't we already have user / company in evaluation context ? Isn't it standard in qweb ?